src/makeRobot.js

```diff
@@ -37,8 +37,60 @@
  *
  * @return {Robot}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
```

Comment:
```suggestion
    name,
```
Use object property shorthand. Since the key of the object and the variable name are the same, you can simplify this to just 'name, version, wheels' instead of 'name: name, version: version, wheels: wheels'.